app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -55,6 +55,7 @@
 public class MainActivity  extends BaseActivity
     implements FragmentManager.OnBackStackChangedListener {
 
+    public static Context contextOfApplication;
```

Comment:
Is it really a good idea to make this public?

Reply:
Because this is the only way I can think of taking MainActivity context out. So I have to do public.